src/relayclient/KnownRelaysManager.ts

```diff
@@ -1,107 +1,102 @@
-import { event2topic } from '../common/utils'
 import RelayRegisteredEventInfo from './types/RelayRegisteredEventInfo'
-import { Address, RelayFilter } from './types/Aliases'
-import { IRelayHubInstance } from '../../types/truffle-contracts'
+import { Address, AsyncScoreCalculator, RelayFilter } from './types/Aliases'
 import RelayFailureInfo from './types/RelayFailureInfo'
 import ContractInteractor from './ContractInteractor'
 import { GSNConfig } from './GSNConfigurator'
+import GsnTransactionDetails from './types/GsnTransactionDetails'
 
 export const EmptyFilter: RelayFilter = (): boolean => {
   return true
 }
+/**
+ * Basic score is reversed transaction fee, higher is better.
+ * Relays that failed to respond recently will be downgraded for some period of time.
+ */
+export const DefaultRelayScore = async function (relay: RelayRegisteredEventInfo, txDetails: GsnTransactionDetails, failures: RelayFailureInfo[]): Promise<number> {
+  const gasLimit = parseInt(txDetails.gas ?? '0')
```

Comment:
1. ?? '0' is prone to bugs imo
2. both gasLimit and gasPrice are txDetails.gas...

Reply:
1. It is called here after the actual default values are set, but the lint's null type check does not know that
2. Fixed